gapic/src/main/com/google/gapid/perfetto/views/MultiSelectionView.java

```diff
@@ -19,22 +19,26 @@
 import static com.google.gapid.widgets.Widgets.createStandardTabItem;
 
 import com.google.gapid.perfetto.models.Selection;
+import com.google.gapid.perfetto.models.Selection.Kind;
 
 import org.eclipse.swt.SWT;
 import org.eclipse.swt.layout.FillLayout;
 import org.eclipse.swt.widgets.Composite;
 import org.eclipse.swt.widgets.TabFolder;
 
+import java.util.NavigableMap;
+
 /**
  * Displays multiple different selections.
  */
 public class MultiSelectionView extends Composite {
-  public MultiSelectionView(Composite parent, Selection[] selections, State state) {
+  public MultiSelectionView(Composite parent, NavigableMap<Kind<?>, Selection<?>> selections,
```

Comment:
Maybe just ask for an `Iterable<Selection<?>>` or `Collection<Selection<?>>`.

Reply:
If I understand it right: I addressed this comment by changing to a more abstract type Map<Kind<?>, Selection<?>>.